build/makefiles/actor.mk

```diff
@@ -91,17 +91,17 @@ start:
 #	$(WASH) ctl update actor  \
 #        $(shell $(WASH) ctl get hosts -o json | jq -r ".hosts[0].id") \
 #	    $(shell make --silent actor_id) \
-#	    $(REG_URL) --timeout 3
+#	    $(REG_URL) --timeout-ms 3000
 
 inventory:
 	$(WASH) ctl get inventory $(shell $(WASH) ctl get hosts -o json | jq -r ".hosts[0].id")
 
 ifneq ($(wildcard test-options.json),)
 # if this is a test actor, run its start method
 # project makefile can set RPC_TEST_TIMEOUT to override default
-RPC_TEST_TIMEOUT ?= 2
+RPC_TEST_TIMEOUT_MS ?= 2000
 test::
-	$(WASH) call --test --data test-options.json --rpc-timeout $(TEST_TIMEOUT) \
```

Comment:
I didn't see other hits for TEST_TIMEOUT, so I assumed this was a bug.

Reply:
Good catch! Thanks!